polaris-service/src/main/java/org/apache/polaris/service/PolarisApplication.java

```diff
@@ -226,7 +230,17 @@ public void run(PolarisApplicationConfig configuration, Environment environment)
         new PolarisCallContextCatalogFactory(
             entityManagerFactory, metaStoreManagerFactory, taskExecutor, fileIOFactory);
 
-    PolarisAuthorizer authorizer = new PolarisAuthorizerImpl(configurationStore);
+    ConcurrentHashMap<RealmContext, EntityCache> realmEntityCache = new ConcurrentHashMap<>();
```

Comment:
Realms are never removed from this map... is that a concern (for large deployments)?

Reply:
The `RealmEntityManagerFactory` already stores the cache in a map that's never cleaned up. I'm hoping to address this with the `@RealmScoped` annotation in the HK2 branch - we could consolidate all the realm-specific caches in a single location with its own read/write-based TTL